cmd/tar2rpm/main.go

```diff
@@ -142,15 +146,32 @@ func main() {
 			Requires:    requires,
 			Conflicts:   conflicts,
 		})
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "tar2rpm error: %v\n", err)
+		os.Exit(1)
+	}
+	if *useDirAllowlist {
+		al := map[string]bool{}
```

Comment:
I think declaring as
var al map[string]bool 

would be better, to avoid possibly unnecessary allocation.